### PR TITLE
rtpengine: possibility to set received-from flag from script

### PR DESCRIPTION
**Pull request by nikbyte**

### CODE REVIEW

File: modules/rtpengine/rtpengine.c [hunk 1]
```diff
@@ -1804,8 +1804,11 @@ static int parse_flags(struct ng_flags_parse *ng_flags, struct sip_msg *msg,
 					err = "missing value";
 					if (!val.s)
 						goto error;
-				}
-				break;
+				} else if (str_eq(&key, "received-from")) {
+					if (val.s)
+						ng_flags->received_from = val;
+				} else
+					break;
```
Comment (wdoekes) [inline modules/rtpengine/rtpengine.c:1811]: Missing `continue`?

Comment (razvancrainea) [reply modules/rtpengine/rtpengine.c:1811]: Yup, will push a fix now.

Comment (razvancrainea) [reply modules/rtpengine/rtpengine.c:1811]: This was actually already fixed by Nick.

Comment (wdoekes) [reply modules/rtpengine/rtpengine.c:1811]: P.S. You guys have gone missing from freenode. Is there a new secret hideout where all the cool kids are? ;-)

**Pull request merged**